### ISSUE TITLE
Counting warning

### ISSUE THREAD

**Issue by Rainbow-Spike** [dokufreaks/plugin-discussion#291]
PHP Warning:  count(): Parameter must be an array or an object that implements Countable in /var/www/comicslate.org/lib/plugins/discussion/action.php on line 425

$cnt = count($data['comments']);

What may be wrong here?

**Comment by Dr-Yukon** [comment]
fastfix
```
if (is_array($data['comments'])) {
	$cnt = count($data['comments']);
} else {
	$cnt = 0;
};

**Comment by Klap-in** [comment]
Fixed by #269
